### PR TITLE
add layer ch.swisstopo.geologie-geowege

### PR DESCRIPTION
**Pull request by rfrancel**

### CODE REVIEW

File: chsdi/locale/empty_chsdi.po [hunk 1]
```diff
@@ -784,6 +784,10 @@ msgstr ""
 msgid "ch.swisstopo.geologie-geotechnik-ziegeleien_1907.ziegelei_2"
 msgstr ""
 
+msgid "ch.swisstopo.geologie-geowege"
+msgstr ""
+
+
```
Comment (AFoletti) [inline chsdi/locale/empty_chsdi.po]: two empty lines instead of one

File: chsdi/models/vector/stopo.py [hunk 2]
```diff
@@ -30,6 +30,19 @@ class GravimetrischerAtlasMetadata (Base, Vector):
...
+    __tablename__ = 'geowege'
+    __table_args__ = ({'schema': 'geol', 'autoload': False})
+    __template__ = 'templates/htmlpopup/geologie_geowege.mako'
+    __bodId__ = 'ch.swisstopo.geologie-geowege'
+    id = Column('bgdi_id', Integer, primary_key=True)
+    titel_1 = Column('titel_1', Text)
+    titel_2 = Column('titel_2', Text)
+    the_geom = Column(Geometry(geometry_type='GEOMETRY',
+                               dimension=2, srid=21781))
+register('ch.swisstopo.geologie-geowege', GeologieGeowege)
```
Comment (AFoletti) [inline chsdi/models/vector/stopo.py]: please put an empty line before "register". The python validation during buildout should have warned you

File: chsdi/locale/empty_chsdi.po [hunk 3]
```diff
@@ -784,6 +784,9 @@ msgstr ""
 msgid "ch.swisstopo.geologie-geotechnik-ziegeleien_1907.ziegelei_2"
 msgstr ""
 
+msgid "ch.swisstopo.geologie-geowege"
```
Comment (cedricmoullet) [inline chsdi/locale/empty_chsdi.po]: And the other translations ?

**Comment by cedricmoullet** [comment]
Is it for next deploy ?

**Comment by rfrancel** [comment]
Yes it is for the next deploy. Other translations are not needed since the term for the tooltip (title) is already translated. Am I correct or did I miss something?

**Comment by AFoletti** [comment]
LGTM

**Pull request merged**